config/initializers/30-redmine.rb

```diff
@@ -29,9 +29,11 @@
 
 I18n.default_locale = 'en'
 # Adds fallback to default locale for untranslated strings
+I18n::Backend::Simple.send(:include, I18n::Backend::Fallbacks)
+I18n.fallbacks.defaults = [I18n.default_locale]
+
 if Setting.table_exists? # don't want to prevent migrations
-  I18n::Backend::Simple.send(:include, I18n::Backend::Fallbacks)
-  I18n.fallbacks.defaults = [I18n.default_locale] + Setting.available_languages.map(&:to_sym)
+  I18n.fallbacks.defaults += Setting.available_languages.map(&:to_sym)
```

Comment:
This will lead to a double entry for English by default, i.e. `[:en, :en, :de, :fr]`. I know this has been like that before your PR but we should use the opportunity to fix it. It's probably not a problem but I'd still prefer it if every language would be in there only once.

I.e. either in line 33:

```ruby
I18n.fallbacks.defaults = Set.new [I18n.default_locale]
```

or in line 36:

```ruby
I18n.fallbacks.defaults = (I18n.fallbacks.defaults + Setting.available_languages.map(&:to_sym)).uniq
```